.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,61 @@
+name: Build and Deploy Docs
+on:
+  push:
+    branches:
+      - master
```

Comment:
there is no master branch anymore. Can be removed.

Reply:
To be honest I'm not quite sure how to handle this. Currently, I see that the main branch does not have this workflow, although it used to have it.
My point was to leave it like this for now, so that we don't trigger build by accident as there's no master branch anymore